html-relative-links/about-me/about-me.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>About Me</title>
+  <style>
+    img {
+      width: 300px;
+    }
+  </style>
+</head>
+<body>
+  <ul>
+    <li><a href="/html-relative-links/index.html">Home Page</a></li>
```

Comment:
This is partially correct. This is definitely a relative path but one problem here is when we change our folder name to `html-relative-links-somethingelse` now all the links are broken. In order to prevent this to happen, we can leverage `../` to navigate around file structure :)